hedera-node/hedera-mono-service/src/main/java/com/hedera/services/store/contracts/EntityAccess.java

```diff
@@ -43,36 +43,17 @@ public interface EntityAccess {
     /* --- Account access --- */
     void customize(AccountID id, HederaAccountCustomizer customizer);
 
-    long getBalance(AccountID id);
-
-    boolean isExtant(AccountID id);
-
-    boolean isUsable(AccountID id);
-
-    boolean isTokenAccount(Address address);
-
-    ByteString alias(AccountID id);
+    boolean isUsable(Address address);
 
     /* --- Storage access --- */
     void recordNewKvUsageTo(
             TransactionalLedger<AccountID, AccountProperty, HederaAccount> accountsLedger);
 
     void putStorage(AccountID id, UInt256 key, UInt256 value);
```

Comment:
If we change key and value to `Bytes` I think we can drop tuweni units.  And whatever changes are needed deeper.